### DISCUSSION TITLE
Has the “/workflow” command been removed in the latest release (v1.160.7)?

### DISCUSSION THREAD

**Discussion by kmrgirish** [zed-industries/zed#20494] [Feature Requests]
I found the `/workflow` command in assistant panel quite useful, but after upgrading to the latest version (v1.160.7), it seems to have been removed. This change was disappointing, as I couldn’t find an alternative feature that provides the same functionality as the “/workflow” command. The experimental suggest edits feature doesn’t appear to be as effective.

Could anyone confirm if the `/workflow` command has indeed been removed in this release, or suggest any alternative solutions that offer similar functionality?

For now I have reverted back to v1.159.10

**Comment by SomeoneToIgnore** [comment]
It has been replaced with a different model to add the edits, see the release notes in https://github.com/zed-industries/zed/pull/19900  for more details.

**Reply by athre0z** [reply]
It's still mentioned on the website here: https://zed.dev/ai. You may want to update that. :)

**Reply by SomeoneToIgnore** [reply]
Thanks, that should be removed now.
